editor/src/clj/editor/gl/vertex2.clj

```diff
@@ -345,55 +270,38 @@
         [expanded-attributes expanded-attribute-locations] (expand-attributes+locations attributes attribute-locations)]
     (gl/gl-bind-buffer gl GL/GL_ARRAY_BUFFER vbo)
     (assign-attributes! gl expanded-attributes expanded-attribute-locations)
+    (gl/gl-bind-buffer gl GL/GL_ARRAY_BUFFER 0)
     expanded-attribute-locations))
 
 (defn- unbind-vertex-buffer-with-shader! [^GL2 gl expanded-attribute-locations]
-  (clear-attributes! gl expanded-attribute-locations)
-  (gl/gl-bind-buffer gl GL/GL_ARRAY_BUFFER 0))
-
-(defn- bind-index-buffer! [^GL2 gl request-id ^IntBuffer index-buffer]
-  ;; TODO: Feed in actual version. Perhaps deftype IndexBuffer with IntBuffer + version?
-  (let [ibo (scene-cache/request-object! ::ibo2 request-id gl {:index-buffer index-buffer :version 0})]
-    (gl/gl-bind-buffer gl GL/GL_ELEMENT_ARRAY_BUFFER ibo)))
-
-(defn- unbind-index-buffer! [^GL2 gl]
-  (gl/gl-bind-buffer gl GL/GL_ELEMENT_ARRAY_BUFFER 0))
```

Comment:
Like in `gl.vertex`, index buffer related functionality was removed. We now the `GLBinding` returned from `gl.attribute/make-index-buffer` to manage index buffers. This was only used from `extension-rive`.